CHANGELOG.md

```diff
@@ -0,0 +1,57 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+For releases `< 1.0.0` minor version steps may indicate breaking changes too.
+
+## [1.0.2] - 2023-10-14
+### Added
+moment.js version updating
+
+## [1.0.1] - 2023-05-22
+### Added
+German translation for Date search and Spatial functions 
+
+## [1.0.0] - 2023-04-28
+### Fixed
+Datarangepicker
+
+## [template] - YYYY-MM-DD
+
+### Added
+
+### Changed
+
+### Removed
+
+### Fixed
+
+### Security
+
+### Deprecated
+
+### Known issues
+
+## [template] - YYYY-MM-DD
+
+### Added
+
+### Changed
+
+### Removed
+
+### Fixed
+
+### Security
+
+### Deprecated
+
+### Known issues
+
+[Unreleased]: https://github.com/tum-gis/ckanext-grouphierarchy-sddi/compare/0.0.5...HEAD
```

Comment:
This needs to be updated to `<LAST RELEASE>`...HEAD`:
https://github.com/tum-gis/ckanext-grouphierarchy-sddi/compare/1.0.2...HEAD